src/zope/dublincore/dcsv.py

```diff
@@ -18,22 +18,16 @@
 
 http://dublincore.org/documents/dcmi-dcsv/
 """
-__docformat__ = 'restructuredtext'
-
 import re
 
 __all__ = "encode", "decode"
 
 try:
-    basestring
+    unicode
 except NameError:
-    # define basestring in Python 2.2.x:
-    try:
-        unicode
-    except NameError:
-        basestring = str
-    else:
-        basestring = str, unicode
+    basestring = str  # PY3
+else:
+    basestring = str, unicode   # noqa: F821
```

Comment:
Let's not try to cater for Python 2.2 any more?  Python 2.7 has basestring, and the old code didn't try to redefine it, and I don't like that the new code does.